packages/node/src/integrations/http/index.ts

```diff
@@ -90,6 +90,22 @@ interface HttpOptions {
    */
   ignoreIncomingRequestBody?: (url: string, request: RequestOptions) => boolean;
 
+  /**
+   * Controls the maximum size of HTTP request bodies attached to events.
+   *
+   * Available options:
+   * - 'none': No request bodies will be attached
+   * - 'small': Request bodies up to 1,000 bytes will be attached
+   * - 'medium': Request bodies up to 10,000 bytes will be attached (default)
+   * - 'always': Request bodies will always be attached
+   *
+   * Note that even with 'always' setting, bodies exceeding 1MB will never be attached
+   * for performance and security reasons.
+   *
+   * @default 'medium'
+   */
+  maxRequestBodySize?: 'none' | 'small' | 'medium' | 'always';
```

Comment:
m: Just to confirm - you're still going to change the name here?

Reply:
ah thanks - seems like I missed this